dataactcore/scripts/updateCGAC.py

```diff
@@ -0,0 +1,46 @@
+from dataactcore.models.jobModels import Submission
+from dataactcore.models.userModel import User
+from dataactcore.models.validationInterface import ValidationInterface
+from dataactcore.models.userInterface import UserInterface
+from dataactcore.models.jobTrackerInterface import JobTrackerInterface
+
+def updateCGAC():
```

Comment:
This is a one-time script to convert our current users/jobs from agency to cgac, yes? What's the standard procedure for something like this? Do we need it in the repo, do we just remove it once the data is converted?